HT4_part1.py

```diff
@@ -0,0 +1,50 @@
+# import modules random and string
+import random
+import string
+
+def random_dicts():
+    j = 1
+    thisdict = {}
+    number_of_keys = random.randint(1, 28)
+    while j <= number_of_keys:
+        thisdict.update({random.choice(string.ascii_lowercase): random.randint(0, 100)})
+        j = j + 1
+    return(thisdict)
+
+def keys_list():

```

Comment:
It's better to have actionable names for functions, use verbs, for example `get_keys_list`

Reply:
Also, let's hand list of dits (for what we need to get keys) over as parameter